enclave-runtime/src/tls_ra/tls_ra_client.rs

```diff
@@ -129,13 +129,17 @@ where
 	}
 
 	/// Reads the payload header, indicating the sent payload length and type.
-	fn read_header(&mut self, start_bytes: Vec<u8>) -> EnclaveResult<TcpHeader> {
+	fn read_header(&mut self, start_byte: u8) -> EnclaveResult<TcpHeader> {
+		const PAYLOAD_SIZE_LENGTH: usize = 8; // u64 indicating the payload size is 8 bytes
+
+		debug!("Read first byte: {:?}", start_byte);
 		// The first sent byte indicates the payload type.
-		let opcode: Opcode = start_bytes[0].into();
+		let opcode: Opcode = start_byte.into();
+		debug!("Read header opcode: {:?}", opcode);
 		// The 8 bytes following afterwards indicate the payload length.
```

Comment:
Remove this comment. It's already not matching the code anymore.

Reply:
But it is still accurate, I would rather keep it for now..

Reply:
But it's not 8 bytes, but a constant defined above. Isn't that a no-go comment because it's likely to go unnoticed if the constant is changed?

Reply:
I see your point - I will update the comment to something less 'brittle'